src/plugins/components/layout/centered-container.js

```diff
@@ -29,6 +29,10 @@ module.exports = function () {
           paddingLeft: 'calc((100% - 1500px)/2)',
           paddingRight: 'calc((100% - 1500px)/2)',
         },
+        '.cc': {
+          paddingLeft: 0,
+          paddingRight: 0,
+        },
```

Comment:
I added both Mike's approach and mine for now so you can see a comparison. This is my approach to support nested centered-container and I think it works as expected if I understand the intent. Also, I think there's some issues with support for browsers that have scroll bar by using vw unit (not obviously on Mac since scrollbar is turned off by default). We can talk more tomorrow @sherakama 

![Screenshot 2023-07-11 at 4 51 27 PM](https://github.com/SU-SWS/decanter/assets/42749717/5bb560c2-122c-4577-a88a-6c2cb731ea1f)